platipy/imaging/label/comparison.py

```diff
@@ -31,11 +31,49 @@ def compute_volume(label):
 
     return sitk.GetArrayFromImage(label).sum() * np.product(label.GetSpacing()) / 1000
 
+def compute_surface_dsc(label_a, label_b, tau=3.0):
```

Comment:
Nice work!
I'm not sure if timing is really an issue, but the distance maps can take a while to compute so an alternative could be to dilate each contour with `sitk.BinaryDilate(a_contour, tau_in_voxel_units)`

Another point to discuss: there is also a function `sitk.BinaryDilate` which seems to do something very similar to `sitk.BinaryContour` - but with different output!

Reply:
Ah yeah good idea. I will have a play with that on Tuesday to see if the dilate works for this :)